### DISCUSSION TITLE
TTGO LORA V1 ohne SD-  Uhrzeit stellen - Display anpassen.

### DISCUSSION THREAD

**Discussion by Lordcybertracker** [cyberman54/ESP32-Paxcounter#748] [General]
Hallo,
ich habe ein paar mehr Baustelle und fragen

    ich habe einen TTGO Lora32 V1
    https://de.aliexpress.com/item/32840238513.html
    Das ist der ganz alte ohne SD Karte.
    Aber egal was ich an Hardware auswähle entweder geht das Display nicht, Läßt sich nicht beschreiben oder Lora verbindet nicht.
    Was muss ich da einstellen oder könnte man das hinzufügen?

    Wie bekomme ich die Uhr auf dem gestellt?

    Wie kann ich das Display änder. Im Laufverkehrsbereich reicht es ja die Zahl ob Joined, TX und Datum mit Uhrzeit anzeigt.
    Danke

**Comment by hansju-11** [comment]
Hello @Lordcybertracker ,
for the problem with time see #760. We have a running solution in dev branch.
